functions/nodejs-apps.bash

```diff
@@ -202,3 +202,147 @@ nodered_setup() {
     dashboard_add_tile "nodered"
   fi
 }
+
+## Function for downloading zigbee2mqtt to current system
+##
+##    zigbee2mqtt_download(String prefix)
+##
+zigbee2mqtt_download() {
+  echo -n "$(timestamp) [openHABian] Downloading zigbee2mqtt... "
+  #if ! [[ -d "${1}/zigbee2mqtt" ]]; then
+    cond_echo "\\nFresh Installation... "
+    if ! cond_redirect mkdir /opt/zigbee2mqtt; then echo "FAILED (mkdir /opt/zigbee2mqtt)";echo1; fi
```

Comment:
Why are you using echo here instead of return? We want to use the return to that we can properly exit and display an error message to the user.

Reply:
correct. I removed the echo. 
I didn't use returns because I didn't want to abord the whole installation if the "git clone" works. If you don't agree I can of course insert the "return 1"

Reply:
Well if git clone fails, then the whole installation will not work, right? If so it makes sense to exit early and then not potentially break the install more by trying to continue.

Reply:
of course: If git clone breaks then stop. But if only mkdir braks we can try to install anyway